Added "group by" and "alias by"

retrodaredevil opened the pull request:
This PR adds options to configure "group by" and "alias by", similar to how you can configure the "Data path".

This is compatible with older installations of this, so upgrading won't break or change anything.

I had to change the structure of the query function to allow for multiple DataFrames to be returned for a single set of docs. (Because you can't do everything with a single DataFrame if you wanted to use group by)

Closes #4

retzkek commented:
Looks really good, thanks! Seems to work as I'd expect. Can you add a little blurb on it to the README?

retrodaredevil commented:
Ok, @retzkek, I just updated the readme and I also changed the inputWidth of each FieldForm to 24 because it was really small for bigger data paths/group bys/alias bys.

retzkek commented:
Awesome, thanks so much!
